packages/@webex/plugin-meetings/src/interpretation/index.ts

```diff
@@ -121,14 +121,31 @@ const SimultaneousInterpretation = WebexPlugin.extend({
    * Update self's interpretation information (self is interpreter)
    * @param {Object} interpretation
    * @param {String} selfParticipantId
-   * @returns {void}
+   * @returns {bool} is target language changed
    */
   updateSelfInterpretation({interpretation, selfParticipantId}) {
+    this.set('selfIsInterpreter', true);
+    const preTargetLanguage = this.targetLanguage;
     const {originalLanguage, sourceLanguage, order, isActive, targetLanguage, receiveLanguage} =
       interpretation || {};
     this.set({originalLanguage, sourceLanguage, order, isActive, targetLanguage, receiveLanguage});
     this.set('selfParticipantId', selfParticipantId);
+
+    return !!(targetLanguage && preTargetLanguage !== targetLanguage);
```

Comment:
in some other PR comment you mentioned that if user wants to disable the translation they should set the target language to 0, but here if targetLanguage is zero, we return false as if "nothing has changed", but disabling interpretation is a change, isn't it? or am I misunderstanding how this works?

also what's the relationship of isActive with targetLanguage being zero, can you have isActive=true and targetLanguage=0?

Reply:
In actual implementation, an interpreter will never allowed be changed to normal attendee, which means he will never have chance to speak in the floor. For example, if host assign one attendee as an interpreter who take charge of "English" to "French", the interpreter's target language will always be English, or be French in verse when he click the toggle button, and never be zero.
Host can set up several interpreters for one channel, e.g. "Engish to French", and hand over one by one. The working person is active.